_guide/_pages/interviews/wrap-up.md

```diff
@@ -56,26 +56,37 @@ Begin by introducing yourself by saying this or something similar to it:
+- ["What has your experience been with user-centered design and/or user research?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.1407do81qrqo)
 
-- ["Please explain a complex technical concept or tool to me as if I were a non-technical partner"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.8lizmty3jisl)
+### Technology and open source
+Ask one or more of the following:
+- ["Are you comfortable with sharing your work in a public fashion?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.hmo4ig69x3qn)
+- ["What is your comfort level with peer-reviewed work?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.hmo4ig69x3qn)
+- ["Your product owner is very excited about a proprietary subscription based product which would cost twice what an open source solution would. How would you encourage the product owner to use open source?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.hmo4ig69x3qn)
 
-- "At 18F we strive to be as representative of the populace as possible and work hard to ensure that everyone understands this notion and is treated fairly and respectfully. [What do diversity, equity, and inclusion mean to you?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.5kwvj8b9hvhd)"
+### Communication and collaboration
+- ["How have you overcome any communication challenges and barriers in a remote/distributed environment? What crucial skills are needed to make a remote first team collaborative?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.3eohpexz765y)
 
-- ["Is there anything that you feel we should have asked you relating to your personal or professional goals that we did not?"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.frpyzn3gj97y)
+- [“Pick a foundational web development concept or technology and explain it at two levels: first as you would to a colleague who’s not a software developer, like a designer or product manager; next, as you would to a peer."](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.d811gtybsywx)
+
+### Diversity and inclusion
+- [“Tell me about a time you’ve worked with or for communities different than your own. What did you learn?”](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.6z03t2jik0y6)"
 
 ### Conclusion
 
+- ["Is there anything that you feel we should have asked you relating to your personal or professional goals that we did not?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.a4271815jmvt)
```

Comment:
I’m _really glad_ this question stayed in. It was consistently my favorite question to ask since it always had such a variety of responses.